packages/arb-token-bridge-ui/src/components/TransferPanel/TokenButton.tsx

```diff
@@ -20,7 +20,7 @@ import { useTokensFromLists, useTokensFromUser } from './TokenSearchUtils'
 
 export type TokenButtonOptions = {
   symbol?: string
-  logoSrc?: string
+  logoSrc?: string | null
```

Comment:
`null` is used if we want to show native currency, otherwise selected token is show as the token logo